brainstormforce/astra
# AST-1541 - Column figure blocks overlapping on one another

### Description
- This seems to be existing issue not specific to 5.9
- As replicated here as well - http://ast-dev.kyllrwmluu-zng4p7p294dp.p.runcloud.link/astra-default-colors/st-template/
- Basically old CSS for figure block affecting these layouts

### Screenshots
- https://d.pr/v/2mhxhV
- Fixed versions - https://share.bsf.io/YEuBZplg

### Types of changes
- Bug fix

### How has this been tested?
- Add default column block (50%-50% layout)
- Add images both the sides
- it will overlapp one another
- By switching to this branch & clearing cache-hard reload this should fix as pre fixed version -  https://share.bsf.io/YEuBZplg

### Checklist:
